lms/djangoapps/certificates/settings.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Languages supported for custom course certificate templates
+LANGUAGES = (
+    ('en', u'English'),
+    ('es', u'Español')  # base Spanish
+)
```

Comment:
I'm still new to the Django-world, but I just want to make sure this is the right location for this. Would it make more sense for this to be a general LMS-setting that can be independently configured in each openedx installation? That way, open-source users have the option to include more languages without having to commit anything to the codebase.

Reply:
Yeah that is definitely a good point.
For your suggestion, would that entail creating a new model, something like Dark Lang, that can be updated from the admin page?

Reply:
I don't think it has to. I think you could add a line to `lms/envs/common.py` to store the supported template languages. Whatever you set in there would be the default for all openedx installations. We have a separate configuration repo that applies to edx.org, where you would add the edx-specific config. Once the config is setup, you can access it in code through the `settings` object.

Reply:
Ok yeah that was my initial approach actually, but for some reason when I added a setting to that file I wasn't able to access it through the settings. Perhaps we can discuss after All Hands.

Reply:
You might need to add it to `lms/envs/aws.py` as well, since doing so will allow us to set edx-specific values via our configuration repo.

Reply:
Do you think this update influences the need for a help_text comment about updating the available certificate template languages?

Reply:
My opinion is that we shouldn't need to say anything about updating available certificate template languages in the help_text, since the help_text is for the person filling in the form, who most likely doesn't have control over that. It would be better to add a comment in the relevant model/form helpers and above the setting itself in the settings file.